flink-connectors/flink-connector-kafka/src/test/resources/log4j-test.properties

```diff
@@ -16,15 +16,14 @@
 # limitations under the License.
 ################################################################################
 
-log4j.rootLogger=INFO, testlogger
+log4j.rootLogger=OFF, testlogger
```

Comment:
Why we should change this?

Reply:
In general, unit test should run silently without log4j logs by default. Otherwise it will pollute the screen and slow down the tests. Failures should be reported via the testing framework such as JUnit/ TestNG. If detail logs are needed for debugging, one should manually turn on the log4j logs.